odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/controller/DataQualityRunsController.java

```diff
@@ -15,9 +16,19 @@ public class DataQualityRunsController implements DataQualityRunsApi {
     private final DataQualityRunsService service;
 
     @Override
-    public Mono<ResponseEntity<DataQualityResults>> getDataQualityTestsRuns(
-            final ServerWebExchange exchange) {
-        return service.getDataQualityTestsRuns()
-                .map(ResponseEntity::ok);
+    public Mono<ResponseEntity<DataQualityResults>> getDataQualityTestsRuns(final List<Long> namespaceId,
+                                                                            final List<Long> datasourceId,
+                                                                            final List<Long> ownerId,
+                                                                            final List<Long> titleId,
+                                                                            final List<Long> tagId,
+                                                                            final List<Long> deNamespaceId,
+                                                                            final List<Long> deDatasourceId,
+                                                                            final List<Long> deOwnerId,
+                                                                            final List<Long> deTitleId,
+                                                                            final List<Long> deTagId,
```

Comment:
Due it is List let's rename args to `namespaceIds`, `datasourceIds` etc.

Reply:
and specification too

Reply:
done